### PR TITLE
Kafka in Kubernetes

### PR DESCRIPTION
**Pull request by gznznzjsn**

### CODE REVIEW

File: kubernetes/task-service/redis/ts-redis-stateful.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: ts-redis-stateful
+spec:
+  serviceName: ts-redis
+  replicas: 1
```
Comment (h1alexbel) [inline kubernetes/task-service/redis/ts-redis-stateful.yaml:7]: set more replicas, 3 is a good choice

File: kubernetes/task-service/task-service-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: task-service
+spec:
+  replicas: 1
```
Comment (h1alexbel) [inline kubernetes/task-service/task-service-deployment.yaml:6]: set more replicas, 3 is a good choice

File: kubernetes/zookeeper/zookeeper-stateful.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: zookeeper-stateful
+spec:
+  serviceName: zookeeper
+  replicas: 1
```
Comment (h1alexbel) [inline kubernetes/zookeeper/zookeeper-stateful.yaml:7]: set more replicas, 3 is a good choice

File: kubernetes/task-service/mongo/ts-mongo-stateful.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: ts-mongo-stateful
+spec:
+  serviceName: ts-mongo
+  replicas: 1
```
Comment (h1alexbel) [inline kubernetes/task-service/mongo/ts-mongo-stateful.yaml:7]: set more replicas, 3 is a good choice

File: kubernetes/order-service/postgres/os-postgres-stateful.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: os-postgres-stateful
+spec:
+  serviceName: os-postgres
+  replicas: 1
```
Comment (h1alexbel) [inline kubernetes/order-service/postgres/os-postgres-stateful.yaml:7]: set more replicas, 3 is a good choice

File: kubernetes/order-service/order-service-deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: order-service
+spec:
+  replicas: 1
```
Comment (h1alexbel) [inline kubernetes/order-service/order-service-deployment.yaml:6]: set more replicas, 3 is a good choice

File: kubernetes/kafka/kafka-deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafka-deployment
+spec:
+  replicas: 1
```
Comment (h1alexbel) [inline kubernetes/kafka/kafka-deployment.yaml:6]: set more replicas, 3 is a good choice

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Comment by gznznzjsn** [comment]
@h1alexbel everything is done here

File: kubernetes/zookeeper/zookeeper-stateful.yaml [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+              valueFrom:
+                configMapKeyRef:
+                  name: zookeeper-configmap
+                  key: ZOOKEEPER_TICK_TIME
+            - name: ZOOKEEPER_SYNC_LIMIT
+              valueFrom:
+                configMapKeyRef:
+                  name: zookeeper-configmap
+                  key: ZOOKEEPER_SYNC_LIMIT
+#      volumes:
```
Comment (h1alexbel) [inline kubernetes/zookeeper/zookeeper-stateful.yaml:38]: @gznznzjsn why do you remove volumes?

Comment (gznznzjsn) [reply kubernetes/zookeeper/zookeeper-stateful.yaml]: I had a problem with it, so I commented it. Now problem is gone, forgot to bring volume back

**Pull request merged**